CollAction/Frontend/app/global/WithinMonthsValidation.ts

```diff
@@ -0,0 +1,52 @@
+import * as moment from "moment";
+
+if (typeof $ !== "undefined" && typeof $.validator !== "undefined" && typeof $.validator.unobtrusive !== "undefined") {
+
+    $.validator.addMethod("withinmonthsaftertoday",
```

Comment:
This would a bit more readable if the string used camel-casing or something similar.

Reply:
Indeed. Done!